chainstate/src/detail/spend_cache/mod.rs

```diff
@@ -241,7 +248,7 @@ impl<'a, S: BlockchainStorageRead> CachedInputs<'a, S> {
         let inputs = tx.inputs();
         let outputs = tx.outputs();
 
-        let inputs_total = self.calculate_total_inputs(inputs)?;
+        let inputs_total = self.calculate_mlt_total_inputs(inputs)?;
```

Comment:
We don't use "mlt" for talking about coins. We called them coins, so we stick to that.

Reply:
Fixed! Thanks!